docs/user/requirements.rst

```diff
@@ -20,7 +20,7 @@ The following packages are needed in order to run RocketPy:
 - Numpy >= 1.0
 - Scipy >= 1.0
 - Matplotlib >= 3.0
-- netCDF4 >= 1.4, < 1.6 for Python 3.7+, netCDF4 >= 1.6.2 for Python 3.11
+- netCDF4 >= 1.4, < 1.6 for Python 3.8+, netCDF4 >= 1.6.2 for Python 3.11
```

Comment:
@MateusStano can you confirm that python 3.8, 3.9 and 3.10 work well with netCDF==1.62 and above? 
Because in the above suggestion it is the expected behavior.